packages/devextreme/js/ui/data_grid.d.ts

```diff
@@ -2303,12 +2303,7 @@ export interface dxDataGridColumnButton<TRowData = any, TKey = any> extends Colu
     name?: DataGridPredefinedColumnButton | string;
     /**
      * @docid dxDataGridColumnButton.onClick
-     * @type_function_param1 e:object
-     * @type_function_param1_field component:dxDataGrid
-     * @type_function_param1_field model:object
-     * @type_function_param1_field event:event
-     * @type_function_param1_field row:dxDataGridRowObject
-     * @type_function_param1_field column:dxDataGridColumn
+     * @type_function_param1 e:{ui/data_grid:ColumnButtonClickEvent}
```

Comment:
Let's make a counterpart fix for TreeList and add doc comments for the ColumnButtonClickEvent type(s) to be published.
Blocked until the mono-repo CI fix.

Reply:
Moreover, TopicsGenerator fails without declaring the "ColumnButtonClickEvent" type linked via uid ref e:{ui/data_grid:ColumnButtonClickEvent}

Reply:
Discard TreeList. Add declarations for DataGrid.